docs/1.getting-started/5.third-parties.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Third Parties
+description: Learn how to optimize the performance of third-party resources using built-in composables and components.
+navigation.icon: i-ph-users-duotone
+---
+
+Nuxt provides a number of composables and built-in libraries that make it easier -- and faster -- to load third-party resources in your application.
+
+- `useScript`: Load any third-party script with server-side rendering support and a proxied API.
+- `useStyle`: Load any third-party stylesheet using various asset strategies.
```

Comment:
I think we can keep these composables as internal in relation to this docs page. End user does not need to know about them, this makes it easier for us to change the lower level implementation in the future if / when we need

Reply:
Also the composable seems to be `useStyles` not `useStyle`

Reply:
I'll wait for Julien's input as well, but fixed the typo for now